docs/earthfile/earthfile.md

```diff
@@ -584,6 +584,28 @@ For more information see the [multi-platform guide](../guides/multi-platform.md)
 
 Same as [`FROM --allow-privileged`](#allow-privileged).
 
+## VERSION
+
+#### Synopsis
+
+* `VERSION [--use-copy-include-patterns] <version-number>`
+
+#### Description
+
+The command `VERSION` identifies which set of features to enable in Earthly while handling the corresponding Earthfile. The `VERSION` command is optional; however if specified, it must be the first command in the Earthfile.
+
+| Version number | enabled features |
+| --- | --- | --- |
+| `0.5` | _initial functionality will be preserved_ |
+
+#### Options
+
+##### `--use-copy-include-patterns`
+
+Instructs Earthly to set buildkit's `include pattern` when executing `COPY` commands; without this feature, Earthly sends all files in the current build context except ignored files.
```

Comment:
We should explain this from the perspective of the user: what is it that the user gains. We should assume that people don't know about BuildKit's internals here.

A better explanation could be "experimental faster COPY algorithm" or something along those lines.